clients/aws-sdk-bedrock-runtime/tests/integration/conftest.py

```diff
@@ -0,0 +1,72 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+from pathlib import Path
+
+import pytest
+from smithy_aws_core.identity.chain import create_default_chain
+from smithy_http.aio.crt import AWSCRTHTTPClient
+
+from aws_sdk_bedrock_runtime.client import BedrockRuntimeClient
+from aws_sdk_bedrock_runtime.config import Config
+
+MODEL_ID = "amazon.titan-text-express-v1"
+BIDIRECTIONAL_MODEL_ID = "amazon.nova-sonic-v1:0"
+
+
+def _create_bedrock_client(region: str):
+    """Helper to create a BedrockRuntimeClient for a given region."""
+    http_client = AWSCRTHTTPClient()
+    return BedrockRuntimeClient(
+        config=Config(
+            endpoint_uri=f"https://bedrock-runtime.{region}.amazonaws.com",
+            region=region,
+            aws_credentials_identity_resolver=create_default_chain(http_client),
+            transport=http_client,
+        )
+    )
+
+
+@pytest.fixture
+def bedrock_client():
+    """Create a BedrockRuntimeClient for us-west-2."""
+    return _create_bedrock_client("us-west-2")
```

Comment:
nit; do we need to make these fixtures or is the helper function sufficient? Setting up a new client per test isn't notable overhead and this may be a premature optimization.

Reply:
I kinda agree with this. It may also make the tests confusing to read for newer developers who don't know about pytest fixtures. It requires they know about fixtures and that they're defined in `conftest.py`. I remember being confused by them when I first started working on botocore